PWGHF/TableProducer/candidateSelectorDstar.cxx

```diff
@@ -0,0 +1,364 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file candidateSelectorDstar.cxx
+/// \brief Selection on D* decay candidates
+///
+/// \author Deependra Sharma <deependra.sharma@cern.ch>, IITB
+/// \author Fabrizio Grosa <fabrizio.grosa@cern.ch>, CERN
+
+// O2
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/Logger.h"
+#include "Framework/runDataProcessing.h"
+// O2Phyics
+#include "Common/Core/TrackSelectorPID.h"
+// PWGHF
+#include "PWGHF/DataModel/CandidateReconstructionTables.h"
+#include "PWGHF/DataModel/CandidateSelectionTables.h"
+#include "PWGHF/Utils/utilsAnalysis.h"
+
+using namespace o2;
+using namespace o2::analysis;
+using namespace o2::framework;
+
+// Struct to applying Dstar selection cuts
+struct HfCandidateSelectorDstar {
```

Comment:
ok will do.

Reply:
Now, will you ask to change file name (candidateSelectorDstar.cxx-->candidateSelectorDstatToD0Pi.cxx)?

Reply:
why not PiD0?  same question for comments below on particle decay.

Reply:
Because `D0Pi` puts the heaviest daughter first, it is consistent with the other names like `BplusToD0Pi` and you also call the decay `D*(+) --> D0 + π(+)` somewhere.